IntegratedChannelEvents.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\ChannelBundle;
+
+/**
+ * @author Jan Sanne Mulder <jansanne@e-active.nl>
+ */
+final class IntegratedChannelEvents
+{
+    /**
+     * The CONFIG_CREATE_REQUEST event is fired when a new config is created.
+     *
+     * This event allows for the request to be intercepted before any mutations are made
+     * in the controller.
+     *
+     * @Event("Integrated\Bundle\ChannelBundle\Event\GetResponseConfigEvent")
```

Comment:
Don't really know if it does anthing but was taking some inspiration from the FOS user bundle (https://github.com/FriendsOfSymfony/FOSUserBundle) where they marked there event this way. Also Symfony self also tags there events except they don't actualy supply the Event class.